src/Lists/OrderedList.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import OrderedList from './OrderedList';
+
+export default {
+  title: 'Components|OrderedList',
+  component: OrderedList
+};
+
+export const Basic = () => (
+  <OrderedList>
+    <li>Apple</li>
```

Comment:
Only a suggestion but since we're forced to use li tags based on the styling, making using them somewhat superfluous, a similar pattern we use in molekule is an items prop:
```js
<OrderedList
  items={[
    'Apple',
    'Dog',
    'Groot'
  ]}
/>
```

Reply:
So if we use an `items` props then we should allow, JSX elements in addition to strings.  If we go that route, then we we can't utilize JSX children pattern.  Should I just use the children prop?

Reply:
@vietnogigoodrx good case for a render prop? 😄 
```jsx
<OrderedList
  renderItem={item => <SomeComponent>{item}</SomeComponent>}
  items={[
    'Apple',
    'Dog',
    'Groot'
  ]}
/>
```

Reply:
That would work, but it's restrictive on `SomeComponent`.  I think im going with the children route since its the least complex and it maintains the parent children pattern.

Reply:
@vietnogigoodrx what do you mean by restrictive?

Reply:
actually, im wrong, its not restrictive since you can add logic in the function

Reply:
@erikshestopal is there any advantage vs

```
<OrderedList>
  <SomeComponent>Apple</SomeComponent>
  <SomeComponent>Dog</SomeComponent>
  <SomeComponent>Groot</SomeComponent>
</OrderedList>
```?

Reply:
There's less code to write since the only variable that's changing is the content and not the component that is rendering the content. 

The convention in our codebase is to use some sort of `items` array with some sort of `renderItem` prop. 

It seems like you don't agree that this is the best pattern to use here.

Reply:
Yea, I dont agree, but it's not a strong disagreement. I will go ahead and add an `items` prop and we can add a `renderItem` prop if we need to in the future.

Reply:
@vietnogigoodrx Would you mind elaborating on why you disagree? The render prop pattern is a widely adopted and trusted pattern by both React core teams and community: https://facebook.github.io/react-native/docs/flatlist#renderitem but I'd love to hear your perspective.

Regardless, we haven't seen how else this component can or should be used so we are likely prematurely optimizing here. Maybe there will only ever be strings as children 🤷‍♀. I'll reiterate my suggestion above but will leave to y'all for what and when to add to molekule:
> ... wonder if a general rule of thumb for adding to molekule should be for design to use it twice.

Reply:
I disagree because the render prop pattern feels like an overkill for the component.  I am comparing it with the `ol`/`ul` components and both of them are quite simple.  They expect an `li` children that are styled.  I was hoping to maintain this pattern for this component.

The reason I am adding it to to the molekule repo based on Devon's suggestion.  She said that we would be using this design in many places.

Reply:
Understood and sounds good regarding intention. I left a couple of other comments but fine with this usage pattern